xmtp_mls/src/builder.rs

```diff
@@ -183,7 +183,7 @@ mod tests {
     async fn legacy_keys() {
         // This test is supposed to be run with a fresh state where account_address has not been registered.
         // Subsequent runs will use a different code path.
-        let account_address = "0x0bD00B21aF9a2D538103c3AAf95Cb507f8AF1B28".to_lowercase();
+        let account_address = "0x0bd00b21af9a2d538103c3aaf95cb507f8af1b28";
```

Comment:
Will this test fail if we leave it with a mix of upper and lower case? It might almost be better to test it without normalizing the address

Reply:
Yes the current use of strings for account addresses is a bit hacky and I'm planning to replace all account addresses with `ethers::types::Address` which is a byte array with necessary type conversion logics. I think it'd be a better canonical type than string or lower case string.

Before we do, I'm ok to leave these as is since they won't be used anyways. wdty?

Reply:
To answer your question, yes the test will fail because we have `to_lowercase()` all over the code base now.